README.md

```diff
@@ -1,5 +1,6 @@
-# eBPF for Windows
+<img src="https://user-images.githubusercontent.com/22178676/158223145-b0fd4151-41e9-4fc3-8b0d-0bf3c37fc772.svg" width=75 height=75 align=left />
```

Comment:
the IMG URL should be something in the ebpf-for-windows repo, so please add the images to this PR.